src/main/kotlin/no/nav/syfo/Apprec.kt

```diff
@@ -15,7 +15,8 @@ data class Apprec(
     val tekstTilSykmelder: String? = null,
     val senderOrganisasjon: Organisation,
     val mottakerOrganisasjon: Organisation,
-    val validationResult: ValidationResult?
+    val validationResult: ValidationResult?,
+    val ebService: String?
```

Comment:
Bør denne ha null som defaultverdi i tilfelle den mangler? For den vil jo mangle på alt fra før du prodsetter mottaksendringen?

Reply:
true